Zend/tests/bug24773.phpt

```diff
@@ -6,7 +6,7 @@ Bug #24773 (unset() of integers treated as arrays causes a crash)
     unset($array["lvl1"]["lvl2"]["b"]);
 ?>
 --EXPECTF--
-Fatal error: Uncaught TypeError: Cannot access offset of type string on string in %s:%d
+Fatal error: Uncaught TypeError: Cannot access offset of type string in unset in %s:%d
```

Comment:
I think this case shows we probably should improve the message somewhat. As this is inconsistent with trying to unset the offset of a string (which is banned in general). Maybe strings do need to be seperate :/